content/spin/kv-store.md

```diff
@@ -157,6 +176,39 @@ command = "npm run build"
 
 {{ blockEnd }}
 
+{{ startTab "Python" }}
+
+```toml
+spin_version = "1"
+authors = ["Dev Eloper <dev.eloper@example.com>"]
+description = "hello"
+name = "hello"
+trigger = { type = "http", base = "/" }
+version = "0.1.0"
+
+[[component]]
+id = "python-sdk-example"
+source = "app.wasm"
+
+# Set an environment variable to tell the code where to fetch the TOML file from:
```

Comment:
I think this and `allowed_http_hosts` are not needed?  Other K/V examples don't use them and I don't see any mention in the code.  Could be wrong though!